docs/getting_started/install_manually.md

```diff
@@ -354,160 +354,141 @@ See [Set up directory permissions](set_up_directory_permissions.md) for more inf
+git clone https://github.com/ezsystems/ezplatform.git
 ```
 
-### 5. Create a new database for eZ Platform
+You can check out a tag, or use the `master` branch if you are interested in working with the latest version.
 
-Create new database (you can substitute `ez1` with the database name you want to use):
+!!! tip
 
-``` bash
-mysql -uroot -ppassword -e "CREATE DATABASE ez1"
-```
+    You can use any other folder name for your project in place of `ezplatform`.
+    Set its location as your project root directory in your Virtual Host configuration.
 
-### 6. Install additional requirements for eZ Platform
+### Install Composer globally
 
-a. Install PEAR/PECL extension:
+Download and run [Composer-Setup.exe](https://getcomposer.org/download/) - it will install the latest Composer version whenever it is executed.
 
-``` bash
-cd c:\program files\php\php.ini
-curl -O https://pear.php.net/go-pear.phar
-php -d detect_unicode=0 go-pear.phar
-php install-pear-nozlib.phar
-pear channel-update pear.php.net
-pecl channel-update pecl.php.net
-pear upgrade-all
-pear config-set auto_discover 1
-```
+#### Install dependencies with Composer
 
-b. Edit `php.ini` and add following line:
+From the folder into which you downloaded eZ Platform, run:
 
 ``` bash
-extension=intl.so
+composer install
 ```
 
-c. Enable opcache extension for PHP (suggested, but not required) by adding:
+Once the installer gets to the point that it creates `app/config/parameters.yml`, you will be presented with a few decisions:
 
-``` bash
-zend_extension=opcache.so
-```
+1. Choose a [secret](http://symfony.com/doc/current/reference/configuration/framework.html#secret); it should be a random string, made up of up to 32 characters, numbers, and symbols. This is used by Symfony when generating [CSRF tokens](http://symfony.com/doc/current/book/forms.html#forms-csrf), [encrypting cookies](http://symfony.com/doc/current/cookbook/security/remember_me.html), and for creating signed URIs when using [ESI (Edge Side Includes)](http://symfony.com/doc/current/book/http_cache.html#edge-side-includes).
+2. You can accept the default options for `database_driver`, `database_host` and `database_port`
+3. For `database_name` and `database_user`, replace them if you customized those values during configuration.
+4. If you set a password for your database user, enter it when prompted for `database_password`.
 
-### 7. Install eZ Platform
+The installer should continue once you've completed this manual portion of the installation process.
 
-a. Download archive from [ezplatform.com](https://ezplatform.com/#download-option). Extract the eZ Platform archive to a directory, then execute post install scripts.
+### Create a new database for eZ Platform
+
+Create new database. Run following command inside MySQL Shell:
 
 ``` bash
-cd /<directory>/
-php -d memory_limit=-1 composer.phar run-script post-install-cmd
+CREATE DATABASE ezplatform CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_520_ci;
```

Comment:
I think it's just utf8 in v1, see https://github.com/ezsystems/developer-documentation/pull/241